_docs/_latest/configure/notifier/index.md

```diff
@@ -241,10 +264,10 @@ To verify violations appear in the Cloud SCC Beta Dashboard, [run the notifier](
 after you have [built an inventory]({% link _docs/latest/use/cli/inventory.md %})
 and [run the scanner]({% link _docs/latest/use/cli/scanner.md %}).
 
-### Email notifications with SendGrid
+### Email notifications
 
-Forseti Security can send email notifications using the SendGrid API. SendGrid
-is the suggested free email service provider for GCP. For information about
+Forseti Security can send email notifications using the SendGrid API as well. 
```

Comment:
I thought we only support sendgrid for this version, do we need the as well at the end?

Reply:
Fixed